pyinfra/operations/server.py

```diff
@@ -276,10 +276,12 @@ def partition(predicate, iterable):
 @operation
 def mount(
     path,
+    device=None,
+    fs_type=None,
     mounted=True,
     options=None,
```

Comment:
```suggestion
    mounted=True,
    options=None,
    device=None,
    fs_type=None,
```

Let's keep the existing order just in case anyone is passing args not kwargs.